tamoc/ambient.py

```diff
@@ -758,7 +770,11 @@ def plot_parameter(self, parm):
         self.ds[parm].plot(y=self.ztsp[0])
         if plt.ylim()[0] <= 0:
             plt.gca().invert_yaxis()
-        plt.tight_layout()
+        with warnings.catch_warnings():
```

Comment:
I don't know why MPL was warning on every call to tight_layout -- googling didn't help. But it was cluttering up my Notebook, so this suppresses it. This does clutter up the code though, so there may be a better solution